.devcontainer/devcontainer.json

```diff
@@ -1,12 +1,23 @@
 {
-  "name": "Development Container for CCF C++ Apps",
+  "name": "Development Container for scitt-ccf-ledger",
   "dockerFile": "Dockerfile",
-  "extensions": [
-    "ms-vscode.cpptools"
-  ],
+  "hostRequirements": {
+    "cpus": 4
+  },
+  "remoteEnv": {
+    "PLATFORM": "virtual"
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-vscode.cpptools"
+      ]
+    }
+  },
   "features": {
     "ghcr.io/devcontainers/features/docker-from-docker:1": {
       "version": "20.10.8"
     }
-  }
+  },
+  "onCreateCommand": "apt-get update && apt-get install -y libcurl4-openssl-dev"
```

Comment:
Should this live in the `Dockerfile` instead? I'm not sure if it makes much difference tbh.

Reply:
The outcome is the same, but yes, it makes more sense to put it in the `Dockerfile`, forgot about that.